### PR TITLE
JFG-896: Replace hardcoded 17 colors for plots with tunable function generateColors() which produce equally distributed colors

### PR DESCRIPTION
**Pull request by anton-antonenko**

### CODE REVIEW

File: dbapi/src/main/java/com/griddynamics/jagger/dbapi/util/ColorCodeGenerator.java [hunk 1]
```diff
@@ -3,85 +3,92 @@
...
+        // These vars are needed for equal distribution of colors
+        final BigDecimal brightnessSteps = new BigDecimal(2);
+        final BigDecimal saturationSteps = new BigDecimal(2);
+        final BigDecimal hueSteps = new BigDecimal(15);
+
+        final BigDecimal brightnessStep = new BigDecimal("0.5").divide(brightnessSteps, 2, RoundingMode.HALF_UP);
+        final BigDecimal saturationStep = new BigDecimal("0.6").divide(saturationSteps, 2, RoundingMode.HALF_UP);
+        final BigDecimal hueStep = ONE.divide(hueSteps, 8, RoundingMode.HALF_UP);
+
+        for (BigDecimal brightness = ONE; brightness.compareTo(new BigDecimal("0.5")) == 1; ) {
```
Comment (a-badaev) [inline dbapi/src/main/java/com/griddynamics/jagger/dbapi/util/ColorCodeGenerator.java]: Those BigDecimal can be cached.

Comment (anton-antonenko) [inline dbapi/src/main/java/com/griddynamics/jagger/dbapi/util/ColorCodeGenerator.java]: Do you mean extracting them to vars?

Comment (a-badaev) [inline dbapi/src/main/java/com/griddynamics/jagger/dbapi/util/ColorCodeGenerator.java]: At least. Or even make them class variables

Comment (anton-antonenko) [inline dbapi/src/main/java/com/griddynamics/jagger/dbapi/util/ColorCodeGenerator.java]: Done

**Comment by a-badaev** [comment]
lgtm

**Comment by dlatnikov** [comment]
👍

**Pull request merged**